### ISSUE TITLE
Find feature not working

### ISSUE THREAD

**Issue by Shad000w** [Beamdog/nwn-issues#381]
**To Reproduce**

 - Go into script editor and open some script.
 - Open Find&Replace feature (CTRL+F) and write some keyword that exists in the script, hit ENTER. Editor will find the first occurence of the searched text and highlight it.
 - Scroll back to the top, re-open Find&Replace and write some different keyword that exist in the script and hit ENTER. Nothing happens.
 - Now, try search the previous keyword and hit ENTER. Nothing happens again.

**Specifics**

 - build no relevant, this is happening for years by now
 - Regression? Yes, not happen in 1.69.
 - OS: Windows 10.

**If needed, describe the bug**

I already reported this previously privately and you said you cannot reproduce it. So I am including a video this time:

https://youtu.be/UfIyt6-qpq8

Simply put, anytime the user changes the keyword, the Find&Replace will fail and do nothing. User has to repeat the input.

Might have to do something with autocomplete which is new feature in EE as it wasn't in 1.69 toolset.

**Comment by pstemarie** [comment]
Followed repro procedure. Can't reproduce on my system - works fine. Subsequently tried all four find/replace function buttons and still can't reproduce. Tested in both .28 Stable and latest Preview. 

Maybe run "verify files" if on Steam or whatever equivalent for other clients.
